src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final Random random = new Random();
+
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        Figure circle = new Circle();
+        Figure rectangle = new Rectangle();
+        Figure rightTriangle = new RightTriangle();
+        Figure square = new Square();
+        Figure isoscelesTrapezoid = new Figure();
+        Figure[] figures = new Figure[]{circle, rectangle, rightTriangle, square, isoscelesTrapezoid};
+        int count = 1;
+        FigureSupplier figure = new FigureSupplier();
+        Figure[] allFigures = new Figure[]{figure.getRandomFigure(figures), figure.getRandomFigure(figures), figure.getRandomFigure(figures),
```

Comment:
The method `getRandomFigure` seems to be expecting an array of figures as a parameter, which is unusual. Typically, the `FigureSupplier` would not require parameters to supply a random figure. Please review the method signature and its usage.